utilities/oadp.py

```
@@ -137,3 +184,173 @@ def create_rhel_vm(
+                status=self.Status.COMPLETED,
+                timeout=self.timeout,
+            )
+        return self
+
+    def __exit__(self, exception_type, exception_value, traceback) -> None:
+        try:
+            if self.teardown:
+                try:
+                    delete_velero_resource(resource=self, client=self.client)
+                except Exception as e:
+                    LOGGER.error(
```

Ruff (0.14.10)

Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)